src/text_classification_pipeline.py

```diff
@@ -126,12 +129,15 @@ def train(self, X_train, y_train) -> None:
 
     def eval(self, X, y) -> None:
         y_pred = self.infer(X)
+        f1 = f1_score(y, y_pred, average=None)
+        self._metrics["f1-score"] = dict((i,j) for i,j in enumerate(f1))
+        self._metrics["f1-score-weighted"] = f1_score(y, y_pred, average="weighted")
+        self._metrics["balanced_accuracy_score"] = balanced_accuracy_score(y, y_pred)
 
-        accuracy = accuracy_score(y, y_pred)
-        f1 = f1_score(y, y_pred, average="weighted")
-
-        print(f"Metrics on test data: \taccuracy = {accuracy}, \tf1_score = {f1}")
+        print("Classification report:")
+        print(self._metrics)
 
+        self.save_results()
```

Comment:
Lets create this bar plots: decision_tree: mean_accuracy, mean_f1; svm: mean_accuracy, mean_f1. And just adding additional report (under bar plot) into Readme about f1 and accuary per classes